arbeidsoppfolging/src/main/kotlin/no/nav/familie/eksterne/kontrakter/arbeidsoppfolging/VedtakOvergangsstønadArbeidsoppfølging.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.familie.eksterne.kontrakter.arbeidsoppfolging
+
```

Comment:
Burde vi bruke enums som typer? Blir litt enklere for konsumenten då? 👀

Reply:
Eller hvordan er "kontraktet" ?

Reply:
Var inne på tanken. Valgte string fordi da bryter ikke kontrakten hvis det skulle være noen endringer, men er åpen for at enums er bedre. På deres side skal de mappe over til lesbar tekst.

Reply:
Gjorde om 👍